Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityCameraSettingsProfile.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.CameraSystem;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality
+{
+    /// <summary>
+    /// Configuration profile for the Windows Mixed Reality Camera settings provider.
+    /// </summary>
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Windows Mixed Reality Camera Settings Profile", fileName = "DefaultWindowsMixedRealityCameraSettingsProfile", order = 100)]
+    [MixedRealityServiceProfile(typeof(WindowsMixedRealityCameraSettings))]
+    public class WindowsMixedRealityCameraSettingsProfile : BaseCameraSettingsProfile
+    {
+        [SerializeField]
+        [Tooltip("Whether to use photo/video camera rendering for Mixed Reality Capture on Windows.")]
```

Comment:
I would add here something to the effect of "if true, this will ensure holograms such as hand tracking align while capturing a video etc etc". I don't think it's clear to any user, except us internally familiar with MRC, that using the PV camera means you will have hologram alignment in your captures

Reply:
That's true. I can add a little more, but I didn't want to overload the tooltip with information and instead added a documentation link next to it to the full platform docs: 

![image](https://user-images.githubusercontent.com/3580640/69374853-d991f500-0c5b-11ea-8b1f-a742d1d3090f.png)

Reply:
Yea the doc button is great, keep that. But we need something else here, short and sweet, that gives more info to the developer for "What is this and why do I give a damn?" 
Hologram alignment during MRC will catch a lot of people's attention & understanding and then they can go learn more.

Reply:
Do you think I should rename the profile entry to be more specific to that, or update the tooltip (or both)?

Reply:
lol I thought of the same consideration for updating the profile entry member name/profile title label

This is my suggestion but feel free to do whatever. Naming is always so tricky

MRC Settings => Mixed Reality Capture Settings
Use PV Camera (Align holograms) [x]       [Documentation button]

Tooltip => something like "If enabled, will turn on PV Camera rendering for MRC with platform. This will ensure that during capture holgorams, such as hand meshes, remain visibly aligned in the video output."

Reply:
Updated, thanks!

Reply:
you can inline the text in the inspector instead of the tooltip... an info callout like we have in boundary

Reply:
Oh good call!